src/detail-page/components/ResourcesList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Accordion } from './accordion/Accordion';
+import { getData } from './ResourcesList.helpers';
+import { ProfessionId } from '../../data/home-page/ITProfessions';
+import { ResourcesType } from '../../data/detail-page/resources/ResourcesInterfaces';
+import { ResourcesTypeNames } from '../../data/detail-page/resources/ResourcesTypeNames';
+
+interface ResourcesListProps {
+  professionId: ProfessionId;
+}
+
+export const ResourcesList: React.FC<ResourcesListProps> = ({
+  professionId,
+}) => {
+  const resourcesData = getData(professionId);
+  return (
+    <div>
+      {Object.keys(resourcesData).map((resourceType, index) => {
+        const resourceTitle = ResourcesTypeNames[resourceType as ResourcesType];
+        const areResourcesAvailable = resourcesData[resourceType as ResourcesType].length !== 0;
```

Comment:
Just to let you know - there is another way to check, if array is not empty - you could write `!!resourcesData[resourceType as ResourcesType].length`, but of course the way you did it is just fine :)